### DISCUSSION TITLE
Does anyone have any good VPS servers that provide high speeds for users in China?

### DISCUSSION THREAD

**Discussion by shadowsocksquestions** [shadowsocks/shadowsocks-rust#927] [Q&A]
i.e. the server would be outside of China...but the users would be within China

I have successfully installed Shadowsocks on a a VPS and can connect to it, however the speed is too slow. The VPS company claims to offer 1Gb/s ports and, when running a speed test from a server in the same country, it was achieving 880Mb/s - which is pretty much what's expected

Using this same VPS and connecting from China, however, speeds will be significantly slower - often in the 20-50Kb/s range. Does anyone know what specifically is causing this - is it because the Chinese Firewall is slowing down traffic out of/into China? If so, how are some other companies/VPS providers able to avoid this (and still get fast - 100Mb/s or more speeds from servers outside of China when connected to by someone from within China?). From what I've read, some companies seem to have a dedicated/shared fibre optic cable into China, hence are able to achieve faster speeds?

Do people have any recommendations for VPS hosts in other countries that will be able to provide an extremely fast connection to a device within China? The current VPS I have is $4 per month and claims to offer a 1Gbps port with "unlimited" (well, 1 month at 1Gbps, possible) bandwidth - I would be looking in that sort of price range, if anyone has any suggestions.

Thanks

**Comment by MarioJames** [comment]
Azure 搬瓦工GIA
